wrappers/wrapper.rb

```diff
@@ -1112,7 +1112,7 @@ def simplify_complex_multi_pickup_or_delivery_shipments(vrp, result = nil, optio
             expanded_services << new_service
           }
 
-          sequence_relations << Models::Relation.create(type: :sequence, linked_ids: sequence_relation_ids)
+          sequence_relations << Models::Relation.create(type: :sequence, linked_ids: sequence_relation_ids.reverse)
```

Comment:
Could you mention in a comment why we do a reverse ?

Reply:
```suggestion
          # For some reason, or-tools performance is better when the sequence relation is defined in the inverse order.
          # Note that, activity.duration's are set to zero except the last duplicated service (so we model exactly same constraint). 
          sequence_relations << Models::Relation.create(type: :sequence, linked_ids: sequence_relation_ids.reverse)
```